discovery-provider/src/trending_strategies/ePWJD_trending_playlists_strategy.py

```diff
@@ -0,0 +1,13 @@
+from src.trending_strategies.base_trending_strategy import BaseTrendingStrategy
```

Comment:
while we aren't actively experimenting with a playlist strategy, let's not add an additional one since we'd be having to compute it in celery